frontend/src/scenes/actions/webhooks/actionWebhooksLogic.ts

```diff
@@ -0,0 +1,126 @@
+import { lemonToast } from '@posthog/lemon-ui'
+import { actions, afterMount, kea, key, listeners, path, props, reducers } from 'kea'
+import { forms } from 'kea-forms'
+import { loaders } from 'kea-loaders'
+import { beforeUnload } from 'kea-router'
+import api from 'lib/api'
+import { toParams } from 'lib/utils'
+
+import { ActionType, HookConfigType } from '~/types'
+
```

Comment:
`Module '"~/types"' has no exported member 'HookConfigType'.`

Also this logic seems to be unreferenced from anywhere else. 🤔

Reply:
Yeah another bad git pick